owl/SOMA-SAY.owl

```diff
@@ -171,8 +159,7 @@ In particular, it defines concepts related to linguistic acts and structures of
 
     <!-- http://www.ease-crc.org/ont/SOMA.owl#AssertionTask -->
 
-    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#AssertionTask">
-        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#IllocutionaryTask"/>
+    <rdf:Description rdf:about="http://www.ease-crc.org/ont/SOMA.owl#AssertionTask">
```

Comment:
It seems like some axioms about assertiontask got into soma-act while others remained in soma-say. Is there a reason for this or should this be refactored?

Reply:
Yes, this is intended: SOMA-SAY declares concepts like DeclarativeCause, but SOMA-ACT does not.